greeklink_core/settings.py

```diff
@@ -196,8 +196,8 @@
     EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
     EMAIL_USE_TLS = True
     EMAIL_HOST = 'smtp.gmail.com'
-    EMAIL_HOST_USER = 'greeklinkverify@gmail.com'
-    EMAIL_HOST_PASSWORD = 'greeklink1'
+    EMAIL_HOST_USER = 'greekrhoverify@gmail.com'
+    EMAIL_HOST_PASSWORD = 'greekrho1'
```

Comment:
Ok yeah this is what I was talking about before, you've already changed these.  Only thing is now instead of keeping them in the code we want to add them to the .env file for local and the AWS environment variables for the cloud.  I can handle the AWS ones if you can add these to the .env file.  Just use the same syntax as the few variables that are already in there, and then check out the other lines of this file that use `os.environ['SETTING_NAME']` for examples on how to get the environment variables